app/Database/Migrations/20220127000000_convertToCI4.php

```
@@ -28,15 +28,17 @@ public function __construct(?Forge $forge = null)
     public function up(): void
     {
         helper('migration');
-        executeScript(APPPATH . 'Database/Migrations/sqlscripts/3.4.0_ci4_conversion.sql');
+        executeScript(APPPATH . 'Database/Migrations/sqlscripts/3.4.0_CI4Conversion.sql');
```

PHPStan (2.2.7)

[error] Constant APPPATH not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)